### PR TITLE
Bootstrap 3 Change

### PR DESCRIPTION
**Pull request by richardhsu**
Updated Naarad for some UI Enhancements:

1. Updated Naarad to generate reports with Bootstrap 3
2. Updated Metric pages to allow linking so you can jump from metric name to its respective graph. (Back to Top also added to navigation bar)
3. Fixed Diff Reports where graphs would not generate due to a change in `naarad.js`.
4. Added some extra quick jump buttons to jump in summary pages.

Example screenshots:

diff-chart
diff-summary
report-chart
report-metrics
report-summary

### CODE REVIEW

**Comment by richardhsu** [comment]
This also addresses bug fix for #294

**Comment by RiteshMaheshwari** [comment]
Cannot automatically merge this.

Also, lets do a demo of the new UI on monday before merging?

**Comment by richardhsu** [comment]
Ah I think the CSV fix breaks the merge but yeah I can demo Monday!



—

File: src/naarad/naarad_constants.py [hunk 1]
```diff
@@ -29,8 +29,8 @@
 # Report Constants
 RESOURCE_PATH = 'resources'
 DEFAULT_REPORT_TITLE = 'naarad analysis report'
-STYLESHEET_INCLUDES = ['bootstrap.min.css', 'inversion.css']
-JAVASCRIPT_INCLUDES = ['jquery-1.7.1.min.js', 'dygraph-combined.js', 'bootstrap.js', 'sorttable.js', 'naarad.js']
+STYLESHEET_INCLUDES = ['bootstrap.css', 'naarad.css']
```
Comment (RiteshMaheshwari) [inline src/naarad/naarad_constants.py]: Any way we can use the minified version? (.min.css usually means minified).

**Comment by RiteshMaheshwari** [comment]
LGTM overall based on as much as I understand these things :)

**Comment by richardhsu** [comment]
Just updated to minified version!

**Comment by RiteshMaheshwari** [comment]
Can you verify everything works after your new change? Once you verify, I will merge.

**Comment by richardhsu** [comment]
Yes, I have built on normal reports and diffs and it is working with the new minified files. 






On Monday, Mar 9, 2015 at 9:55 AM, Ritesh Maheshwari <notifications@github.com>, wrote:

Can you verify everything works after your new change? Once you verify, I will merge.

**Pull request merged**